interfaces.go

```diff
@@ -13,13 +13,16 @@ import (
 	"time"
 )
 
-// eventType is the type of an error event.
+// eventType is the type of error event.
 const eventType = "event"
 
-// transactionType is the type of a transaction event.
+// transactionType is the type of transaction event.
 const transactionType = "transaction"
 
-// checkInType is the type of a check in event.
+// logType is the type of log event.
+const logType = "log"
```

Comment:
For later, but let's maybe turn this into a struct, with the item type and its content type.